x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/grouping/Bucket.java

```diff
@@ -78,52 +76,64 @@ public class Bucket extends EsqlScalarFunction implements Validatable {
         Rounding.builder(TimeValue.timeValueMillis(10)).build(),
         Rounding.builder(TimeValue.timeValueMillis(1)).build(), };
 
+    private static final ZoneId DEFAULT_TZ = ZoneOffset.UTC; // TODO: plug in the config
+
     private final Expression field;
-    private final Expression buckets;
+    private final Expression bucket;
```

Comment:
Inconsistent - the `@Param` and method refer to `buckets` instead - pick either bucket or buckets and use that everywhere.
Also - Bucket.bucket?

Reply:
Reverted back to `buckets`, that was the intention, thx.